conode/Dockerfile-dev

```diff
@@ -10,4 +10,4 @@ COPY exe/conode.Linux.x86_64 ./conode
 
 EXPOSE 7770 7771
 
-CMD [ "./run_nodes.sh", "-n 1" ]
+CMD ["./run_nodes.sh", "-n", "1"]
```

Comment:
Will it work if it's not the full path? 
> If you want to run your command without a shell then you must express the command as a JSON array and give the full path to the executable. [1](https://docs.docker.com/v17.09/engine/reference/builder/#cmd)

Reply:
Yes, it takes the `WORKDIR` into account; and that makes a single place to change when one wants to run it as non-root.

Reply:
Fix for docker documentation in https://github.com/docker/docker.github.io/issues/9025